pkg/cosigner/image_signer.go

```diff
@@ -0,0 +1,116 @@
+package cosigner
```

Comment:
seems like this package could be called "cosign" to line up with the notary package naming

Reply:
done, resolving